### PR TITLE
Minor fixes to moodpage and app

### PR DESCRIPTION
**Pull request by NFranklin14**

### CODE REVIEW

File: app/screens/landing/moodpage/MoodPage.jsx
Comment (blake-Hansen) [inline app/screens/landing/moodpage/MoodPage.jsx:1]: Reminder to team to change axios url for demo

File: app/screens/landing/moodpage/MoodPage.jsx [hunk 1]
```diff
@@ -137,12 +40,27 @@ const styles = StyleSheet.create({
 });
 
 function MoodPage({ navigation }) {
+  const [cuisines, setCuisines] = useState([]);
+
+  useEffect(() => {
+    const helpFunction = async () => {
+      try {
+        const response = await axios.get('http://192.168.1.39:8080/api/v1/cuisines');
```
Comment (fmirando) [inline app/screens/landing/moodpage/MoodPage.jsx:48]: URL should be in a .env in the future

File: app/screens/landing/moodpage/MoodButton.jsx
Comment (blake-Hansen) [inline app/screens/landing/moodpage/MoodButton.jsx:1]: Reminder that user.id is shorthand in this navigation, just marking incase we need to use later.

File: app/App.jsx
Comment (blake-Hansen) [inline app/App.jsx:1]: Initial Route name needs to be changed, this was left for dev purposes. Will continue with merge.

**Review by blake-Hansen — APPROVED** [review]

**Review by fmirando — APPROVED** [review]
Looks good. In the future, we're all going to need to store the URL in a env variable since you're using a different url for your iphone

**Pull request merged**